src/components/editor/editor-window/editor-window.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../node_modules/codemirror/lib/codemirror.css';
+@import '../../../../node_modules/codemirror-one-dark-theme/one-dark.css';
+
+.CodeMirror {
+    height: 100%;
+}
```

Comment:
```suggestion
}

```

Reply:
@SuperSandro2000  You mean the newline at the EOF?

Reply:
Yes. I thought I did not send the comment.